rangeproof/proof.go

```diff
@@ -268,6 +261,17 @@ func (statement *Statement) ProofStructure(index int) (*ProofStructure, error) {
 	return NewProofStructure(index, statement.Sign, statement.Factor, statement.Bound, statement.Splitter)
 }
 
+func (typ StatementType) Sign() int {
+	switch typ {
+	case GreaterOrEqual:
+		return 1
+	case LesserOrEqual:
+		return -1
+	default:
+		return 0
```

Comment:
why not directly throw here an error if the sign should never be anything else but 1 or -1?

Reply:
Well, because the API would become uglier; this function does not return an error. But that reason is not good enough; I'll change it.